src/blockchain/lending/liquidityPool.spec.ts

```diff
@@ -0,0 +1,29 @@
+import test from 'ava'
+
+import { provider } from '../../constants.spec'
+
+import {
+  getLPTBaseValue,
+  getLPTValue,
+  getOptimalUtilizationRate,
+} from './liquidityPool'
+
+// cSpell:disable
```

Comment:
Let's not disable cSpell since we're trying to maintain the spell check throughout the project and add the missing words to `.cspell.json`